engine/class_modules/sc_shaman.cpp

```diff
@@ -7461,11 +7466,19 @@ struct earthquake_overload_damage_t : public earthquake_damage_base_t
   earthquake_overload_damage_t( shaman_t* player, earthquake_base_t* parent ) :
     earthquake_damage_base_t( player, "earthquake_overload_damage", player->find_spell( 298765 ), parent )
   {
-    // Earthquake modifier is hardcoded rather than using effects, so we set the modifier here
-    spell_power_mod.direct = 0.3195 * player->talent.mountains_will_fall->effectN( 1 ).percent();
-    // TODO: implement spellpower coefficient extracttion from spell variable
-    // auto spell_desc = player->dbc->spell_desc_vars( this->data().id() );
-    // spell_desc.desc_vars()
+
+      if ( auto vars = player->dbc->spell_desc_vars( this->data().id() ).desc_vars() )
```

Comment:
Pretty sure this is not going to work. Spells 298765 for overload (and the real one 77478) do not contain the Spell Description Variable you are looking for. It's contained in the base earthquake spell, which would be 61882 or 462620.